akka-stream/src/main/scala/akka/stream/io/Timeouts.scala

```diff
@@ -168,4 +173,13 @@ object Timeouts {
     }
   }
 
+  private final val failureAsCompletionFlow = Flow[Any].transform(() ⇒ new PushStage[Any, Any] {
```

Comment:
> Timeouts.asCompletion is currently not used, I thought it should have been but then it's the wrong timeout I guess, the Timeout stages should throw a custom one. Happy to delete or adjust it.

https://github.com/akka/akka/pull/18770#issuecomment-150613936

Yeah, I'll remove it. I don't think it's wrong per-se, but not something we need to provide (nor is used).